agent/05_agent_create_cluster.sh

```diff
@@ -102,9 +102,60 @@ function wait_for_cluster_ready() {
   echo "Cluster is ready!"
 }
 
+function wait_for_operator() {
+    subscription="$1"
+    namespace="${2:-}"
+    echo "Waiting for operator ${subscription} to get installed on namespace ${namespace}..."
+
+    for _ in $(seq 1 60); do
+        csv=$(oc -n "${namespace}" get subscription "${subscription}" -o jsonpath='{.status.installedCSV}' || true)
+        if [[ -n "${csv}" ]]; then
+            if [[ "$(oc -n "${namespace}" get csv "${csv}" -o jsonpath='{.status.phase}')" == "Succeeded" ]]; then
+                echo "ClusterServiceVersion (${csv}) is ready"
+                return 0
+            fi
+        fi
+
+        sleep 10
+    done
+
+    echo "Timed out waiting for csv to become ready!"
+    return 1
+}
+
+function create_lso_operator_v4_11() {
+  if [[ ${OPENSHIFT_RELEASE_STREAM} == "4.12" || ${OPENSHIFT_RELEASE_STREAM} == "4.13" ]]; then
+    oc delete subscription local-storage-operator -n openshift-local-storage
+    cat > "${mceManifests}/lso_operator_4_11.yaml" << EOF
```

Comment:
I'd remove the 4_11 from the name of this file so it be made more generic if you need to this same mechanism for future releases